toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/tests/logic/test_xnvme_tests_scc.py

```diff
@@ -0,0 +1,106 @@
+"""
+This is a port of the tests:
+xnvme_tests_scc_idfy.sh                 --> test_idfy()
+xnvme_tests_scc_scopy_async.sh          --> test_scopy()
+xnvme_tests_scc_scopy_sync.sh           --> test_scopy_clear()
+xnvme_tests_scc_scopy_msrc_async.sh     --> test_scopy_msrc()
+xnvme_tests_scc_scopy_msrc_sync.sh      --> test_scopy_msrc_clear()
+xnvme_tests_scc_support.sh              --> test_support()
+
+Note: the tests were named "async" did not seem to have any async be-instrumentation.
```

Comment:
This line is a bit confusing, I think this is an improvement
`Note: the tests named "async" did not seem to have any async be-instrumentation`

Reply:
removed / fixed.